ast/file.go

```diff
@@ -215,7 +212,7 @@ func NewImportNode(keyword *KeywordNode, public *KeywordNode, weak *KeywordNode,
 	if semicolon == nil {
 		panic("semicolon is nil")
 	}
-	numChildren := 3
+	numChildren := 2
```

Comment:
I think this needs to be reverted. There are 3 children in the normal case: keyword, name, and semicolon.